mock/datasets/nighttime-lights.data.mdx

```diff
@@ -12,6 +12,16 @@ media:
 thematics:
   - covid-19
   - agriculture
+taxonomy:
+  Topics:
```

Comment:
This makes me wonder if users will understand what they type here is what they will see. (Since this is the only capitalized attribute in mdx, it looks wrong somehow in mdx context.) But I also don't really see a better solution since there will be various requirements for capitalization 🤷

Reply:
Agree capitalized keys (perhaps also with special characters and spaces) could perhaps be avoided, to keep YAML keys clean?

Is this better?

```yaml
taxonomy:
  - title: Topic
    values:
      - Covid 19
      - Agriculture
    - title: Sector
      values:
        - Electricity
```

Reply:
yeah, this is a very valid concern. In this way it perhaps increases clarity with the cost of verbosity (in an already complex frontmatter). Not sure which one is preferable.

@hanbyul-here The filters get encoded as json deliberately. It is not possible to create individual keys for each taxonomy, since we don't know how many there are. Each key is created with a hook, and dynamically creating them would mean to have a loop creating the hooks, which is not ok. Therefore, all taxonomy options are stored as an object under `taxonomy` which get encoded as json. We could write a custom `hydrate|dehydrate` function to make it prettier, but I'm wondering if it is worth it. Thoughts?

Reply:
Oh, my point was more about
When you click 'Ccovid 19' from the card, the URL changes to: http://localhost:9000/discoveries?taxonomy={%22Topics%22:%22ccovid-19%22}
but when you select 'Ccovid 19' from the dropdown, the URL changes to: http://localhost:9000/discoveries?taxonomy=%7B%22Topics%22%3A%22ccovid-19%22%7D

I think it is because JSON.stringify doesn't encode `{}`? I don't think it is a blocker but thought you might have missed it.

Reply:
@hanbyul-here I had completely misunderstood what you meant. 😅 Good catch. Fixed!

Reply:
Yay thanks for handling. I like @j08lue 's idea about making the value explicit. It is a valid concern that it will be verbose, but I feel like that is more coherent with other attributes of metadata too.